changelogs/changelog.yaml

```diff
@@ -124,3 +124,16 @@ releases:
         name: junos
         namespace: null
     release_date: '2020-06-23'
+  1.0.1:
+    changes:
+      bugfixes:
+      - Make `src`, `backup` and `backup_options` in junos_config work when module
+        alias is used (https://github.com/ansible-collections/junipernetworks.junos/pull/83).
+      - Update docs after sanity fixes to modules.
+      minor_changes:
+      - Add changelogs for 1.0.0 release (https://github.com/ansible-collections/junipernetworks.junos/pull/70)
```

Comment:
The entry is not required. This got generated because the changelog PR for 1.0.0 had a changelog entry

Reply:
And the line in CHANGELOG, and the fragment `70-add-change-log-1.0.0.yaml`.